src/pages/HomePage.js

```diff
@@ -29,11 +37,11 @@ const HomePage = () => {
           pr: 2,
         }}
       >
-        <Search items={categories} sx={{ maxWidth: "800px" }} />
+        <Search items={arrayOfCategories} sx={{ maxWidth: "800px" }} />
```

Comment:
pass in the constructed `arrayOfCategories` now instead of the `categories` directly from state